### DISCUSSION TITLE
Azure Data Studio - suggestions with trailing ```

### DISCUSSION THREAD

**Discussion by select-star-from** [community/community#64864] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot v1.82.15 in Azure Data Studio.
Code suggestions often include a trailing ```.
(I believe this marks an end of file in github?)

For me, this is never desired when writing SQL. 
Always needs to be deleted before running the query.
Would be grand if it could be suppressed.

This is my only complaint with an otherwise magnificent tool.
Thank you.

**Comment by nftushar** [comment]
Update: Ensure you're using the latest versions of Copilot and Azure Data Studio, as updates often fix issues.

Settings: Check if there are options in either tool to control or suppress specific types of suggestions.

Feedback: Provide feedback to the developers about this behavior to potentially influence future updates.

Manual Adjustment: Until resolved, manually delete the trailing ``` before running queries.

For the latest solutions, refer to official documentation and community resources for Copilot and Azure Data Studio.

**Reply by select-star-from** [reply]
^The dark side of AI. Before long, everything is just AI bots pretending to be people talking to AI bots pretending to be people.

**Reply by CHabrock** [reply]
This has been happening to me. You ever resolve this issue?

**Reply by select-star-from** [reply]
Sadly, no. I think you're the real first person to even notice this post. (thanks)

**Comment by Jaber-Saed** [comment]
Hello,

Thank you for providing your feedback regarding Copilot in Azure Data Studio. It's great to hear that you find Copilot to be a useful tool, and your input about the code suggestions is valuable.

You mentioned that code suggestions often include a trailing ```, which can be problematic when writing SQL queries. I understand that this can be an inconvenience, as the trailing backticks might need to be removed before running the query. You're correct that in certain contexts, such as Markdown files on GitHub, triple backticks are used to indicate code blocks.

Your suggestion to have an option to suppress the trailing backticks in code suggestions is certainly valid. Providing users with more control over the formatting of suggestions can enhance the user experience, especially in cases where the default suggestion might not align with the intended usage.

I'll make sure to pass along your feedback to the appropriate team responsible for Copilot development. They will appreciate hearing your thoughts and insights on how to improve the tool further. Your feedback is essential in helping shape the direction of tools like Copilot to better suit users' needs.

Thank you again for taking the time to share your thoughts, and I'm glad to hear that Copilot has been generally beneficial to your workflow. If you have any more feedback or suggestions in the future, please don't hesitate to reach out.

Best regards,
Jaber

**Comment by vindelbo** [comment]
Same issue 
![image](https://github.com/community/community/assets/6838752/3203fd23-5a78-4546-9877-91eba879c3e0)

**Comment by ghost** [comment]
Me too.

**Comment by Mjongeling07** [comment]
Here too, issue persists. Since I code a lot in SQL this becomes quite of an annoyance.
